software/nosql/src/main/java/brooklyn/entity/nosql/couchbase/CouchbaseClusterImpl.java

```diff
@@ -229,8 +229,13 @@ protected void doStart() {
                 }
                 
                 ((CouchbaseNode)getPrimaryNode()).rebalance();
-            } else if (getQuorumSize()>1) {
-                log.warn(this+" is not quorate; will likely fail later, but proceeding for now");
+            } else {
+                if (getQuorumSize()>1) {
```

Comment:
Should this really be `if (getQuorumSize() > serversToAdd.size())`? I won't let that block a merge of the PR though.